bindata/network/multus/002-rbac.yaml

```diff
@@ -63,3 +63,48 @@ subjects:
 - kind: ServiceAccount
   name: multus
   namespace: openshift-multus
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels: 
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: multus:view
+rules:
+- apiGroups: ["apiextensions.k8s.io"]
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["k8s.cni.cncf.io"]
+  resources: ["*"]
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels: 
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+  name: multus:edit
```

Comment:
This authorization is wrong. We only want cluster administrators to be able to create CRDs and network-attachment-definitions, and they already have that permission.